src/routes/Projects/components/Projects.tsx

```diff
@@ -24,11 +25,19 @@ export default class Projects extends React.PureComponent<IProjectProps, {}> {
     this.props.getProjects();
     customPageView(window.location.pathname + window.location.search);
   }
+
+  public handleImageError(e: any) {
```

Comment:
This can go outside of the class, as a normal function

Reply:
Should I add it in any of the helpers?

Reply:
Hm, considering that this image is specific for this component I would suggest to add it at the top of this file (before the class declaration)